tests/tom-swifty-test/tom-swifty-test.csproj

```diff
@@ -131,40 +34,2109 @@
     <Folder Include="DylibBinderTests\" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\Dynamo\Dynamo.csproj">
-      <Project>{D0E2964B-34C9-4FEE-A638-A10E2E4A0917}</Project>
-      <Name>Dynamo</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\SwiftReflector\SwiftReflector.csproj">
-      <Project>{65E8BE11-4D82-4EA5-9D14-3CA044D443EF}</Project>
-      <Name>SwiftReflector</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\tom-swifty\tom-swifty.csproj">
-      <Project>{8CAC7366-9650-440D-A3C5-36D880285DD5}</Project>
-      <Name>tom-swifty</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\leaktest\leaktest.csproj">
-      <Project>{33D1E7E3-A123-43F0-B047-163C6E75D3BF}</Project>
-      <Name>leaktest</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\SwiftRuntimeLibrary\SwiftRuntimeLibrary.csproj">
-      <Project>{B7E6CF5A-B836-41CF-988C-A83607AF5445}</Project>
-      <Name>SwiftRuntimeLibrary</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\DylibBinder\DylibBinder.csproj">
-      <Project>{2BB83AA6-D9DB-432F-8FC1-7AC83E7FE97B}</Project>
-      <Name>DylibBinder</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
+    <ProjectReference Include="..\..\Dynamo\Dynamo.csproj" />
+    <ProjectReference Include="..\..\SwiftReflector\SwiftReflector.csproj" />
+    <ProjectReference Include="..\..\tom-swifty\tom-swifty.csproj" />
+    <ProjectReference Include="..\..\leaktest\leaktest.csproj" />
+    <ProjectReference Include="..\..\SwiftRuntimeLibrary\SwiftRuntimeLibrary.csproj" />
+    <ProjectReference Include="..\..\DylibBinder\DylibBinder.csproj" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="SwiftRuntimeLibrary.Mac.dll.config">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Mono.Options" Version="6.12.0.148" />
+    <PackageReference Include="NUnit" Version="4.0.1" />
+    <PackageReference Include="NUnit.ConsoleRunner" Version="3.16.3" />
+    <PackageReference Include="NUnit.Extension.NUnitProjectLoader" Version="3.7.1" />
+    <PackageReference Include="NUnit.Extension.NUnitV2Driver" Version="3.9.0" />
+    <PackageReference Include="NUnit.Extension.NUnitV2ResultWriter" Version="3.7.0" />
+    <PackageReference Include="NUnit.Extension.TeamCityEventListener" Version="1.0.9" />
+    <PackageReference Include="NUnit.Extension.VSProjectLoader" Version="3.9.0" />
+    <PackageReference Include="NUnit.Runners" Version="3.12.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
Not sure what this next ItemGroup accomplishes. Why are all these files being removed from the project? If it is correct, please convert to globbing patterns to reduce the noise.